explorer/main.go

```diff
@@ -69,16 +73,16 @@ func getBasePath(h http.Header) string {
 }
 
 // Setup the router and handle the incoming request.
-func serve(w http.ResponseWriter, r *http.Request) {
-	user, pass, err := GetCredentials()
+func serve(w http.ResponseWriter, r *http.Request, logger *log.Logger) {
```

Comment:
do we need to pass logger like this here? I would think the logger we declared above should work